.circleci/config.yml

```
@@ -657,7 +657,7 @@ defaults:
       name: t_native_test_ext_prb_math
       project: prb-math
       binary_type: native
-      image: cimg/node:18.11
+      image: cimg/rust:1.66.1
```

```suggestion
      image: cimg/rust:1.66
```

Can we just use the latest image here? Does something like `cimg/rust`, `cimg/rust:latest` or `cimg/rust:stable` work?

I think you based this on what we do for `node` but the only reason we're not using simply `cimg/node:18` is that CircleCI at some point dropped those tags and have not restored them so far (https://github.com/CircleCI-Public/cimg-node/issues/130#issuecomment-786051150, https://github.com/CircleCI-Public/cimg-node/issues/294).

---

> Or, better, can we just use the latest image here? Does something like `cimg/rust`, `cimg/rust:latest` or `cimg/rust:stable` work?

The problem is that CircleCI does not have an alias for that. See https://circleci.com/developer/images/image/cimg/rust
While for [nodejs](https://circleci.com/developer/images/image/cimg/node) you can use `cimg/node:current`.

But I will update to `cimg/rust:1.70` at least.